undistro/zora
# Support for JWT Authorization and token refresh

## Description
Add support for services secured using JWT Authorization

## Linked Issues

## How has this been tested?

- Create initial tokens using `helm zoraauth` plugin
- Install into cluster via helm, increasing `refreshThreshold` parameter to speed up refresh (e.g. 23h45m if token valid for 24h)
- Monitor secret values, checking token content to ensure it's valid
  Content should be JSON, containing `accessToken`, `refreshToken` and `token_type`.  The `accessToken` can be checked by accessing the `/userinfo` endpoint for the authorization domain, or viewing in a decoder such as [jwt.io](https://jwt.io/).
- Check invocations against a secure service work as expected
 

## Checklist
